src/main/fc/fc_init.c

```diff
@@ -270,6 +266,22 @@ void init(void)
     ensureEEPROMContainsValidData();
     readEEPROM();
 
+#if defined(STM32F4) && !defined(DISABLE_OVERCLOCK)
+    // If F4 Overclocking is set and System core clock is not correct a reset is forced
+    if (systemConfig()->cpu_overclock && SystemCoreClock != 240000000)
+    {
```

Comment:
Can you put the opening braces at the end of the line with the statement please? (For statements, as opposed to function definitions, see 'K & R style'.)